utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -28,42 +29,68 @@
 import sys
 import docker
 import collections
+from oscap_docker_python.oscap_docker_util_noatomic import OscapDockerScan
+
+atomic_loaded = False
+
+
+class AtomicError(Exception):
+    """Exception raised when an error happens in atomic import
+    """
+    def __init__(self, message):
+        self.message = message
+
 
 try:
     from Atomic.mount import DockerMount
     from Atomic.mount import MountError
     import inspect
 
     if "mnt_mkdir" not in inspect.getargspec(DockerMount.__init__).args:
-        sys.stderr.write(
+        raise AtomicError(
             "\"Atomic.mount.DockerMount\" has been successfully imported but "
             "it doesn't support the mnt_mkdir argument. Please upgrade your "
             "Atomic installation to 1.4 or higher.\n"
         )
-        sys.exit(1)
 
     # we only care about method names
     member_methods = [
         x[0] for x in
         inspect.getmembers(
-            DockerMount, predicate=lambda member: inspect.isfunction(member) or inspect.ismethod(member)
+            DockerMount, predicate=lambda member:
+                inspect.isfunction(member) or inspect.ismethod(member)
         )
     ]
 
     if "_clean_temp_container_by_path" not in member_methods:
-        sys.stderr.write(
+        raise AtomicError(
             "\"Atomic.mount.DockerMount\" has been successfully imported but "
             "it doesn't have the _clean_temp_container_by_path method. Please "
             "upgrade your Atomic installation to 1.4 or higher.\n"
         )
-        sys.exit(1)
+
+    # if all imports are ok we can use atomic
+    atomic_loaded = True
 
 except ImportError:
     sys.stderr.write(
         "Failed to import \"Atomic.mount.DockerMount\". It seems Atomic has "
         "not been installed.\n"
     )
-    sys.exit(1)
+
+except AtomicError as err:
+    sys.stderr.write(err.message)
+
+''' print state of atomic import '''
+if atomic_loaded:
+    print("Atomic seems well installed and ready for use")
```

Comment:
```suggestion
    print("Using Atomic API")
```

Reply:
Yes but if the option --disable-atomic is enabled, we shouln't print this message, so i suggest to say "Using Atomic API" (or Docker Api) but only before creating OscapDockerScan/OscapAtomicScan class. 

I was printing that here because my atomic install was wrong, so it was well loaded but not working. But for a production use I agree that your proposed message is enough.

Reply:
"Production" environment might be broken as well. And it could be helpful even when we explicitly provided `--disable-atomic`, because it might not affect the execution (e.g. not as the first argument).

Reply:
@evgenyz ahh i think i know why we disagree, it's because the place where you proposed this message was just after the imports (so this message was always printed, even if the docker api was used). But you cannot knew that since this part is not in diff anymore !

So I think you will agree to add this message in oscap-docker.in (at the place where we call OscapDockerScan or OscapAtomicScan), or in the constructor of these two classes (but it seems less proper to me).